gallery/src/components/NxToast/NxToastPage.tsx

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxCode,
+  NxP,
+  NxTable,
+  NxTile,
+  NxH2,
+  NxH3,
+  NxTextLink,
+  NxStatefulAccordion,
+  NxAccordion,
+  NxWarningAlert
+} from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile } from '../../gallery-components/GalleryTiles';
+import { GalleryTileFooter } from '../../gallery-components/GalleryTileFooter';
+
+import CodeExample from '../../CodeExample';
+
+const NxToastComplexLayoutExampleCode = require('./NxToastComplexLayoutExample?raw'),
+    NxToastSimpleLayoutExampleCode = require('./NxToastSimpleLayoutExample?raw');
+
+const NxToastPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        Toasts are rendered on the top right corner of the screen, and will remain in place until the user
+        dismisses them individually. Each toast renders one
+        {' '}<NxTextLink href="#/pages/Alert"><NxCode>NxAlert</NxCode></NxTextLink>, which comes in four
+        variations: Error, Info, Warning, and Success.
+      </NxP>
+      <NxP>
+        Toasts generally fall under a "global and static" context, in a sense that throughout the application,
+        toasts are rendered within the same container on the top right corner of the screen.
+      </NxP>
```

Comment:
I'm assuming toasts are only supported in Global Sidebar layouts, since that's all we have examples of? That constraint should be documented.

Reply:
> I'm assuming toasts are only supported in Global Sidebar layouts, since that's all we have examples of? That constraint should be documented.

Actually, it works just fine without the sidebar. I can remove the sidebar from "NxToastSimpleLayoutExample" to show a simpler and cleaner page. If we want to demonstrate Legacy Layouts without the sidebar, I would need to add a header in as well. Would you prefer to have a 3rd example demonstrating Legacy Layouts, or include the header in this instance, and remove it from "NxToastComplexLayoutExample" to demonstrate positioning with/without the header?

Reply:
> Actually, it works just fine without the sidebar. I can remove the sidebar from "NxToastSimpleLayoutExample" to show a simpler and cleaner page

Having a Global Sidebar layout that doesn't actually have a global sidebar isn't one of our valid layouts.

Regarding the legacy page layouts, if we want to support those we first need design to define how that should look – namely whether the toasts in that case should be aligned to the side of the viewport or the side of `NxPageMain` (which are not the same in that case).

Reply:
Please see https://gallery.sonatype.dev/#/pages/Page%20Layout%20Examples for a full understanding of the differences between legacy and global sidebar layouts.

Reply:
> Regarding the legacy page layouts, if we want to support those we first need design to define how that should look – namely whether the toasts in that case should be aligned to the side of the viewport or the side of `NxPageMain` (which are not the same in that case).

Ah, I see what you mean. I was looking through layouts instead of reading that the new GlobalSidebarLayout content area will span the viewport unlike Legacy Layouts. I'll check with design on this.

Reply:
> Regarding the legacy page layouts, if we want to support those we first need design to define how that should look – namely whether the toasts in that case should be aligned to the side of the viewport or the side of `NxPageMain` (which are not the same in that case).

I discussed with design about legacy layouts and where toasts should go in that case, and it was decided that in legacy layouts that the toasts would remain flush with the viewport instead of the content. A legacy layout example was created here: [9c05518](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/9c05518fc4f0e5bcdfa1577d3c5be1f1856fc14c), and the NxToastContainer has now been made a direct child of nx-page instead of NxPageMain.